src/components/cron.ts

```diff
@@ -0,0 +1,22 @@
+import { updateSuggestionCron } from './suggestions';
+import { TextChannel } from 'discord.js';
+import { CommandoClient } from 'discord.js-commando';
+import { CronJob } from 'cron';
+
+const NOTIF_CHANNEL_ID_OLD: string = process.env.NOTIF_CHANNEL_ID_OLD || '.';
+
+// Checks for new suggestions every min
+export const createSuggestionCron = (client: CommandoClient) =>
+  new CronJob('0 */1 * * * *', async function () {
+    const isSuggestionsChanged = await updateSuggestionCron();
+    if (isSuggestionsChanged) {
+      const messageChannel = client.channels.cache.get(NOTIF_CHANNEL_ID_OLD);
+      if (messageChannel == undefined) {
```

Comment:
we can just do a `if (!messageChannel)` here

Reply:
I think i need to account for undefined no matter what

Reply:
does `!messageChannel` not handle undefined?

Reply:
don't think it does

Reply:
that doesn't make sense `!undefined` returns `true` so it should work

Reply:
even if you need to check for undefined specifically, you should use strict equality `===`

Reply:
because `!undefined` returns `true` i shouldn't use `!msgCh` because i can't send msgs to an undefined channel

Reply:
Doesn’t this if statement check if it’s a bad channel? So true is what you want? that’s the same as == undefined

Reply:
Message channel can be `undefined` or a `Channel`. Mhm, true is what I want, but not by `!undefined`.

![image](https://user-images.githubusercontent.com/37851975/124395230-efd5b300-dcd0-11eb-9fd9-8ea363044921.png)
![image](https://user-images.githubusercontent.com/37851975/124395272-20b5e800-dcd1-11eb-9144-e3e456f34914.png)

Reply:
oh bruh ok i think i get what u mean - i thought u meant I dont need an else-if

Reply:
oh no LOL